src/prefect/server/api/concurrency_limits_v2.py

```diff
@@ -266,3 +296,71 @@ async def bulk_decrement_active_slots(
         )
         for limit in limits
     ]
+
+
+ONE_HOUR = 60 * 60
+
+
+@router.post("/active_owners", status_code=status.HTTP_200_OK)
```

Comment:
It took me a second to convince myself that this wouldn't overlap with the `/{id_or_name}` route above; since this one is a `POST` and the other is a `GET`, I think we're fine.  It's a _little_ confusing, but I realized that's what we're doing with like `/flow_runs/filter`, etc.

Reply:
Yeah, it seems to distinguish this and other routes due to the POST vs. GET.